USAGE.md

```diff
@@ -122,3 +122,42 @@
 # plot pdf of exponential distribution 
 >>> exponential.plot_bar_pdf() 
 ```
+
+## For Gamma Distribution
+```
+>>> from probdists import Gamma
+
+# By default Gamma will create a gamma dist. with k=2, theta=2 and won't fit to data
+# This is useful if you want a simple Gamma distribution to play around with the pdf
+# HOWEVER: It will not make use of data passed into the distribution
+>>> gamma = Gamma()
+
+# To use of the sample data or your own data, and approximate a gamma fit to that data:
+>>> gamma = Gamma(fit=True, data_file='demo_gamma_data')
+# for your own file, replace 'demo_gamma_data' with 'my_data_file.txt' 
+# Ensure there is no extra whitespace at end of file
+# The sample data will fit k=2 (rounded to integer), theta~=2.37
+
+# The above is IMPORTANT. 
+# If you don't specify fit=true the Gamma distribution won't fit 
+# but will model using default or inputted k & theta
+# If you specify fit, Gamma distribution will fit and disregard any k,theta argument
+# To provide data for fitting a new Gamma instance will need to be made
+
+# to calculate mean:
+>>> print(gamma.calculate_mean())
+
+# to calculate standard deviation:
+>>> print(gamma.calculate_stddev())
+
+# to calculate pdf, call function and give argument x
+>>> print(gamma.pdf(x))
+0.18165
+
+# to access data 
+>>> print(gamma.data)
+[1, 2, 2, 3, 3, 4, 5, 6, 8, 9, 13]
+
+# plot pdf of exponential distribution 
+>>> gamma.plot_bar_pdf() 
+```
```

Comment:
Everything else looks fine here, except that you've coded up the `__add__()` magic method, and haven't documented here. Could you go ahead and do that as well?